frontend/src/views/ClientListView.vue

```diff
@@ -2,19 +2,20 @@
 import Button from "primevue/button";
 import Card from "primevue/card";
 import InputText from "primevue/inputtext";
+import { onMounted } from "vue";
 import { ref } from "vue";
 
-const clientsList = ref([
-  { id: 1, name: "Artem Kovshov" },
-  { id: 2, name: "Sergej Herbert" },
-  { id: 3, name: "Artem Kovshov" },
-  { id: 4, name: "Daria Eliseeva" },
-  { id: 2, name: "Sergej Herbert" },
-  { id: 3, name: "Artem Kovshov" },
-  { id: 4, name: "Daria Eliseeva" },
-]);
+import { getClients } from "@/backendClient";
+import type { Client } from "@/global/types/entities";
+import { ROUTES } from "@/router";
+
+const clientsList = ref<Client[]>([]);
 
 const value = ref(null);
+
+onMounted(async () => {
+  clientsList.value = await getClients();
```

Comment:
I added pagination recently. This method now returns a paginated client list (the typescript return type needs to be updated).

We can fix that now or later.